docs/src/main/asciidoc/_client-guide/jboss-ejb-client.adoc

```diff
@@ -50,6 +53,12 @@ client context, should use pass-by-reference (instead of pass-by-value) semantic
 * `outbound-connection-ref` Reference to an outbound connection configured in the remoting subsystem.
 * `connect-timeout` The timeout, in milliseconds, to be used while creating a connection.
 
+[[http-connections]]
+=== <http-connections>
+
+* `http-connection` HTTP Connection for EJB invocation.
```

Comment:
`http-connection` should be documented as a sub-element in its subsection under `http-connections`, just like `remoting-ejb-receiver` is a subelement of `ejb-receivers`.